src/main/scala/com/ironcorelabs/IronSdkAdvanced.scala

```diff
@@ -7,4 +7,13 @@ trait IronSdkAdvanced[F[_]] {
 
   def documentEncryptUnmanaged(data: Array[Byte], options: DocumentEncryptOpts): F[DocumentEncryptUnmanagedResult] =
     documentEncryptUnmanaged(ByteVector.view(data), options)
+
+  def documentDecryptUnmanaged(encryptedData: ByteVector, encryptedDeks: ByteVector): F[DocumentDecryptUnmanagedResult]
```

Comment:
We could new type wrap these to make it less likely that the arguments will be passed in the wrong order. We have this situation in other place (and in other languages), but it struck me again so I figured I'd suggest it.

Reply:
Done